src/main/java/edu/texas/social_computing/hospitals/Resident.java

```diff
@@ -33,4 +42,24 @@ public Resident getPartner() {
         return null;
     }
 
+    public boolean hasPartner() {
+        return this.partner != null;
+    }
+
+    /**
+     *
+     * @return the Resident that is matched this a Hospital that is lower on their preference list than their partner.
+     * The tie always goes to this
+     */
+    public Resident worsePlacedPartner() {
```

Comment:
I would probably prefer to see this as a static method `worsePlacedResident(Resident r1, Resident r2)`

Reply:
you got it. that was the way i did it first. i changed it to this when i realized we would never need to do it for anyone that is not partners.